crates/qt-build-utils/src/lib.rs

```diff
@@ -232,6 +234,22 @@ impl QtBuild {
             }
         }
 
+        fn determine_if_has_framework_libs(qmake_executable: &String) -> bool {
+            let lib_path = QtBuild::static_qmake_query(qmake_executable, "QT_INSTALL_LIBS");
+            let target: Result<String, env::VarError> = env::var("TARGET");
```

Comment:
do we need the type defined here? can't it just be `let target = env::var("TARGET");` and let the compiler determine the type?

Reply:
Of course, it's inferred. I tend to type variables when in unknown code.
```suggestion
            let target = env::var("TARGET");
```